modules/task/README.md

```diff
@@ -55,6 +58,20 @@ The async task module is a singleton and a service can initialize
 only one at this time. Users are free to define their own backends
 and encodings for message passing.
 
+It is possible to enqueue and execute tasks on different clusters if you want
+to separate workloads. The default configuration spins up workers that will
+execute the task. In order to spin up just task enqueueing without execution,
+initialize the module as follows:
+
+```go
+  svc, err := service.WithModule(
+    task.New(newBackend, task.DisableExecution()),
```

Comment:
Hmm, should this be taken care of by module roles instead of a one-off for the task module? E.g. shouldn't they just start the service with roles="yarpc" and not "task" if they don't want to start the task module?